src/common/reducers/repositories.js

```diff
@@ -1,42 +1,117 @@
+import merge from 'lodash/merge';
+import union from 'lodash/union';
+
 import * as ActionTypes from '../actions/repositories';
-import { parseGitHubRepoUrl } from '../helpers/github-url';
+
+// XXX move to entities.js
+// XXX should we be handling snaps here too?
+// no, because it's not part of the normalized result from github
+// so this should really be called github_entities
+// no, decompose instead ...
```

Comment:
I want to make the entities reducer it's own module/folder in reducers/, and then it can be split up and import helpers for the various functions.